java/client/src/main/java/glide/api/RedisClusterClient.java

```diff
@@ -342,6 +343,30 @@ public CompletableFuture<String> flushall(
                 FlushAll, new String[] {mode.toString()}, route, this::handleStringResponse);
     }
 
+    @Override
+    public CompletableFuture<String> flushdb() {
+        return commandManager.submitNewCommand(FlushDB, new String[0], this::handleStringResponse);
+    }
+
+    @Override
+    public CompletableFuture<String> flushdb(@NonNull FlushMode mode) {
```

Comment:
This function and the above function are duplicates that are in `RedisClient`. Can `flushdb()` and `flushdb(@NonNull FlushMode mode)` just be moved to `BaseClient`?

Reply:
The review checklist mentions keyless commands with policies should have distinct APIs for cluster and standalone clients, so maybe AWS wants to avoid having the common implementation in BaseClient for some reason (maybe to make it clearer to document?). It seems like we've been following this pattern of distinct standalone/cluster APIs for commands like this so far

Reply:
Unfortunately, no. We have to duplicate this, as it happens with all keyless commands. Please see Aaron's description.
`flushdb` has different docs though in different clients.
On standalone redis it flushes the current DB (server has up to 16).
On cluster it flushes the only DB which server has (so it is equal to `flushall`).

Reply:
You're welcome to continue discussion on https://github.com/aws/glide-for-redis/pull/1593